ISHPermissionKit/Requests/ISHPermissionRequestNotificationsLocal.h

```diff
@@ -15,10 +15,12 @@
  *  @sa ISHPermissionCategoryNotificationLocal
  */
 @interface ISHPermissionRequestNotificationsLocal : ISHPermissionRequest
+
 /**
  *  Settings to configure the notification.
  *
  *  @sa ISHPermissionCategoryNotificationLocal
  */
-@property (nonatomic) UIUserNotificationSettings *noticationSettings;
+@property (nonatomic, nonnull) UIUserNotificationSettings *notificationSettings;
```

Comment:
This obviously breaks backwards compatibility, but it's a trivial fix.

I don't think it either justifies a 2.0 release nor adding a separate property and deprecating the old, which would have the potential of new bugs...